.github/workflows/profiles.yml

```diff
@@ -42,7 +42,7 @@ jobs:
         run: python3 -m pip install jsonschema
       - uses: lukka/get-cmake@latest
       - name: Configure/Generate
-        run: cmake -S. -B build -D UPDATE_DEPS=ON -DCMAKE_BUILD_TYPE=${{matrix.config}} -G "Ninja"
+        run: cmake -S. -B build -D PROFILES_BUILD_TESTS=OFF -D UPDATE_DEPS=ON -DCMAKE_BUILD_TYPE=${{matrix.config}} -G "Ninja"
```

Comment:
@juan-lunarg this is weird to me, we do need to build the profiles tests on C.I.

Reply:
Does testing work on MacOS?

Reply:
I am pretty sure @lunarpapillo enabled it rescently.